syntax = "proto3";

package raft.date;

service RaftDate
{
    rpc GetDateDiffInDays(DateDiffRequest) returns (DateDiffResponse) {}
}

message DateDiffRequest
{
    int64 startDate = 1;
    int64 endDate = 2;
}

message DateDiffResponse
{
    int64 diffInDays = 1;
}